src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.subway.ui;
+
+import nextstep.subway.application.LineService;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import nextstep.subway.dto.LineUpdateRequest;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
```

Comment:
@RequestMapping 을 활용해볼 수도 있겠네요 
```suggestion
@RestController
@RequestMapping("/lines")
```

Reply:
클래스 레벨에서 중복되는 url을 매핑해서 메소드에서 url중복을 피할 수 있는데 
full url을 사용하면 특정 api에서 에러가 났을 때 검색이 용이하기 때문에 
중복이 주는 불편함보다는 검색의 편리함을 선택해서 매핑을 했습니다!